components/cli/pkg/ballerina/docker_bal_executor.go

```diff
@@ -272,3 +275,107 @@ func (balExecutor *DockerBalExecutor) Version() (string, error) {
 func (balExecutor *DockerBalExecutor) ExecutablePath() (string, error) {
 	return "", nil
 }
+
+func (balExecutor *DockerBalExecutor) Init(projectDir string) error {
+	cliUser, err := user.Current()
+	if err != nil {
+		return fmt.Errorf("error while retrieving the current user, %v", err)
+	}
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("error in determining working directory, %v", err)
+	}
+	err = util.CleanAndCreateDir(filepath.Join(currentDir, constants.TARGET_DIR_NAME))
+	if err != nil {
+		return err
+	}
+	dockerCmdArgs := []string{"-u", cliUser.Uid,
+		"-l", "ballerina-runtime=" + version.BuildVersion(),
+		"--mount", "type=bind,source=" + filepath.Join(currentDir, constants.TARGET_DIR_NAME) + ",target=/home/cellery/tmp",
+		"--mount", "type=bind,source=" + projectDir + ",target=/home/cellery/" + projectDir,
+		"-w", "/home/cellery/",
+		"wso2cellery/ballerina-runtime:" + version.BuildVersion(),
+	}
+	dockerCommand := []string{"./" + constants.BalInitTestExecFIle, cliUser.Uid, filepath.Base(projectDir)}
+	dockerCmdArgs = append(dockerCmdArgs, dockerCommand...)
+	var bashArgs []string
+	bashArgs = append(bashArgs, "run")
+	bashArgs = append(bashArgs, dockerCmdArgs...)
+	cmd := exec.Command("docker", bashArgs...)
+	cmd.Env = os.Environ()
+	cmd.Stdout = os.Stdout
+	cmd.Stdin = os.Stdin
+	cmd.Stderr = os.Stderr
+	err = cmd.Run()
+	if err != nil {
+		return fmt.Errorf("error occurred while initializing tests using ballerina docker image, %v", err)
+	}
+	balProjectName := filepath.Base(projectDir) + "_proj"
+	err = util.CopyDir(filepath.Join(currentDir, constants.TARGET_DIR_NAME, balProjectName), filepath.Join(currentDir, balProjectName))
+	if err != nil {
+		return err
+	}
+	err = util.RemoveDir(filepath.Join(currentDir, constants.TARGET_DIR_NAME))
```

Comment:
Change the constant TARGET_DIR_NAME format from snake case to camel case.

Reply:
Fixed